spec/lib/import_csv_to_big_query_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'import_csv_to_big_query'
+require 'rails_helper'
+
+RSpec.describe ImportCSVToBigQuery do
+  let(:table) { double('table').as_null_object }
+  let(:loaded_job) { double('loaded job').as_null_object }
+  let(:dataset) { double('dataset').as_null_object }
+  let(:bigquery_client) { double('Bigquery Client', dataset: dataset) }
+  let(:export_tables) { class_double(ExportTablesToCloudStorage, TABLES: %w[TestTable]) }
+
+  it 'loads the dataset from ENV' do
+    allow(ENV).to receive(:[]).with('CLOUD_STORAGE_BUCKET')
+    expect(ENV).to receive(:[]).with('BIG_QUERY_DATASET').and_return('test')
+    subject.load(bigquery: double('BigqueryClient').as_null_object)
+  end
+
+
+  it 'loads the bucket endpoint from ENV' do
+    expect(ENV).to receive(:[]).with('CLOUD_STORAGE_BUCKET').and_return('test').at_least(:once)
+    allow(ENV).to receive(:[]).with('BIG_QUERY_DATASET')
+    subject.load(bigquery: double('BigqueryClient').as_null_object)
+  end
+
+  it 'connects to the correct data set' do
```

Comment:
I'm not sure this is asserting that it connects to the correct dataset, I'd imagine this and `loads the dataset from ENV` be combined?

Similar to:
` expect(bigquery_client).to receieve(:dataset).with('test_dataset') `

At the moment, the test to me is more `it 'connects to a data set'`.